### PR TITLE
init: website

### PR DESCRIPTION
**Pull request by tristancamejo**
Initiate the initial boilerplate for a next.js website.

### CODE REVIEW

File: components/Banner.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+              <span className="flex p-2 rounded-lg bg-indigo-800">
+                <SpeakerphoneIcon
+                  className="h-6 w-6 text-white"
+                  aria-hidden="true"
+                ></SpeakerphoneIcon>
+              </span>
+              <p className="ml-3 font-medium text-white truncate">
+                <span className="md:hidden"> We need your help! </span>
+                <span className="hidden md:inline">
+                  The future of Evie is currently being made and we need your
```
Comment (thehairy) [inline components/Banner.tsx]: Really now

File: pages/_document.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { Head, Html, Main, NextScript } from "next/document";
+
+export default function Document() {
+  return (
+    <Html data-theme="evie">
```
Comment (thehairy) [inline pages/_document.tsx]: Data theme evie? You wanna sell me something here?

File: tailwind.config.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  },
+  variants: {
+    extend: {
+      animation: ["hover", "focus"]
+    }
+  },
+  daisyui: {
+    themes: [
+      {
+        evie: {
```
Comment (thehairy) [inline tailwind.config.js]: boi, name it different D:

File: components/Footer.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+      label: "Privacy Policy",
+      href: "/privacy"
+    },
+    {
+      label: "Terms of Service",
+      href: "/terms"
+    },
+    {
+      label: "Imageing Community",
+      href: "https://imageing.org/discord"
```
Comment (thehairy) [inline components/Footer.tsx:26]:
```suggestion
      href: "https://imageing.org/9UK5ZcY6By"
```

File: pages/index.tsx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+          <div className="max-w-md">
+            <Image
+              className="hover:scale-110 transition-all duration-500 ease-in-out hover:drop-shadow-2xl"
+              src="/assets/Mark.png"
+              alt="Splash"
+              width={200}
+              height={200}
+            />
+            <p className="py-6">
+              some app to do with images, i wasn't given info. (request changes
```
Comment (thehairy) [inline pages/index.tsx]: Read the fucking repo readme

**Review by thehairy — CHANGES_REQUESTED** [review]

File: pages/_document.tsx (hunk 2 above)
Comment (tristancamejo) [reply pages/_document.tsx]: it's the evie color scheme for daisyui.

File: components/Banner.tsx (hunk 1 above)
Comment (tristancamejo) [reply components/Banner.tsx]: forgot to delete that, oops

Comment (tristancamejo) [reply components/Banner.tsx]: https://github.com/ImageingApp/Website/commit/6243a5fbe1c3c37847abecea20f3078a37e7df27

File: pages/index.tsx (hunk 5 above)
Comment (tristancamejo) [reply pages/index.tsx]: https://github.com/ImageingApp/Website/commit/edb72a06e48b0d94b9ed95bebb0aee6c1bf654c6

File: tailwind.config.js (hunk 3 above)
Comment (tristancamejo) [reply tailwind.config.js]: ok

File: components/branding/FeatureGrid.tsx [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+export default function FeatureGrid() {
+  return (
+    <div className="py-12">
+      <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
+        <div className="lg:text-center">
+          <p className="mt-2 text-3xl leading-8 font-extrabold tracking-tight sm:text-4xl">
+            Features
+          </p>
+          <p className="mt-4 max-w-2xl text-xl text-gray-400 lg:mx-auto">
+            request changes here for whatever you want to be the sub
```
Comment (thehairy) [inline components/branding/FeatureGrid.tsx]: readme bonk

**Review by thehairy — CHANGES_REQUESTED** [review]

File: components/branding/FeatureGrid.tsx (hunk 6 above)
Comment (tristancamejo) [reply components/branding/FeatureGrid.tsx]: https://github.com/ImageingApp/Website/pull/1/commits/bab9ec8e23cb2104b55e1f9c19e48355dc965c26

**Review by thehairy — APPROVED** [review]

**Pull request merged**